cmake/EnableCoverage.cmake

```diff
@@ -0,0 +1,68 @@
+# ~~~
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ~~~
+
+# For the GCC and Clang compiler families, enable a Coverage build type.
+if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "^(Clang|GNU)$")
+
```

Comment:
Do we need/like the blank line after "if" here and below?

Reply:
Maybe, cmake-format used to be stupid about formatting the comments all wrong without these extra lines. Let me try again with a newer version and fix them (in a separate PR).